### PR TITLE
Added Laravel template

### PR DESCRIPTION
**Pull request by roberto-butti**
Added Laravel template #116

### CODE REVIEW

**Comment by roberto-butti** [comment]
The goal of the template is to setup a minimal environment where you can launch Unit and Feature tests in Laravel using phpunit.
The template allow you to:
- run a environment with ubuntu latest version
- checkout your code in the environment
- create a minimale .env file starting from the classical .env.example file included in Laravel
- create a key useful  for running a Laravel app
- create a empty sqlite database
- execute the tests via phpunit injecting a couple of environment variables in order to use the sqlite database created


Some consideration:
- i didn't include the migrate command because to avoid a double migration if you use the trait RefreshDatabase in your tests.
- you could avoid to have the step with db creation if you use Database in memory, for example:
`DB_DATABASE: ":memory:"`

**Comment by exodusanto** [comment]
Tested. Works like a charm 
Zero configuration 🚀

File: ci/laravel.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+name: Laravel CI
```
Comment (andymckay) [inline ci/laravel.yml]: Let's remove the word `CI` as per the guidelines here #156

File: ci/properties/laravel.properties.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "name": "Laravel",
+    "description": "Test a Laravel project.",
+    "iconName": "php",
+    "categories": [
+        "Php",
```
Comment (andymckay) [inline ci/properties/laravel.properties.json]: Should this be `PHP`?

File: ci/laravel.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+on:
+  push:
+    branches:
+    - master
+    - features/*
```
Comment (andymckay) [inline ci/laravel.yml]: As per the guidelines here #156 let's remove any specific branches filtering for a push. Users will be able to add that if they want to.

**Comment by andymckay** [comment]
Thanks @roberto-butti 🙇 because this takes zero-config I think it makes a bit more sense to have this one, just left a few changes in comment.

**Comment by roberto-butti** [comment]
> Thanks @roberto-butti 🙇 because this takes zero-config I think it makes a bit more sense to have this one, just left a few changes in comment.

Thank you for your feedback. I'm going to taking in charge...

File: ci/laravel.yml (hunk 1 above)
Comment (roberto-butti) [reply ci/laravel.yml]: removed CI 
https://github.com/actions/starter-workflows/pull/138/commits/9f247c9a93ece1ceb61bfaae2c1b2b1e273560fa

File: ci/properties/laravel.properties.json (hunk 2 above)
Comment (roberto-butti) [reply ci/properties/laravel.properties.json]: In categories I have PHP now.
https://github.com/actions/starter-workflows/pull/138/commits/9f247c9a93ece1ceb61bfaae2c1b2b1e273560fa

File: ci/laravel.yml (hunk 3 above)
Comment (roberto-butti) [reply ci/laravel.yml]: I'm using on: [push] now

https://github.com/actions/starter-workflows/pull/138/commits/9f247c9a93ece1ceb61bfaae2c1b2b1e273560fa

**Comment by roberto-butti** [comment]
Hi, do you have feedbacks on this Pull Request?

Thank you!

**Comment by andymckay** [comment]
Thank you @roberto-butti 🙇

**Pull request merged**